dvc/repo/experiments/executor/base.py

```diff
@@ -248,6 +250,19 @@ def on_diverged_ref(orig_ref: str, new_rev: str):
             )
         return refs
 
+    @classmethod
+    def _validate_remotes(cls, dvc: "Repo", git_remote: Optional[str]):
+
+        if git_remote == dvc.root_dir:
+            logger.warning(
+                f"'{git_remote}' points to the current Git repo, experiment "
```

Comment:
```
                f"'{git_remote}' points to the current Git repo, experiment "
```

I'm not sure this is helpful since a valid remote would also point to the current Git repo, right? Maybe something like "local workspace" makes more sense than "current Git repo"?

Reply:
The local workspace is your current Git repo. `https://github.com/my/repo.git` is not your current Git repo - it's *a* remote Git repo, and potentially where your current Git repo is cloned from, but "current repo" would always be your current local workspace repo